src/sentry/integrations/jira_server/webhooks.py

```diff
@@ -71,9 +74,8 @@ def post(self, request: Request, token, *args, **kwargs) -> Response:
             handle_assignee_change(integration, data)
             handle_status_change(integration, data)
         except (ApiError, ObjectDoesNotExist) as err:
-            extra.update({"token": token, "error": str(err)})
```

Comment:
I removed token from the extra since it's filtered in Sentry and I don't think it provides any useful data anyway.